package.json

```diff
@@ -24,7 +24,7 @@
         "@rollup/plugin-typescript": "^11.1.6",
         "@symfony/stimulus-testing": "^2.0.1",
         "@vitest/browser": "^2.1.1",
-        "clean-css-cli": "^5.6.2",
```

Comment:
The `cli` version of this package was not necessary, `clean-css` provides a JavaScript API that we can use in our `build_package.js` script.

Reply:
Me looking at this package:

![yw8i_b](https://github.com/user-attachments/assets/d2b5fbc6-bdc3-4434-895c-69d3e621765c)